src/handler.js

```diff
@@ -1,15 +1,19 @@
-"use strict";
+("use strict");
 const fetch = require("node-fetch");
 const { coursesUrl } = require("./utils/constructFunnelbackUrls");
+const { logEntry } = require("./utils/logEntry");
 const { success, error } = require("./utils/format");
 const { transformResponse } = require("./utils/transformResponse");
+const { LOG_TYPES, HTTP_CODES } = require("./constants/constants.js");
 
 module.exports.courses = async (event) => {
     try {
         const requestParams = event.queryStringParameters;
 
         if (!requestParams || !requestParams.search) {
-            return error("The search parameter is required.", 400, "Bad Request", event.path);
+            const errorDetails = { message: "The search parameter is required." };
+            console.error(logEntry(event, HTTP_CODES.BAD_REQUEST, LOG_TYPES.ERROR, errorDetails));
```

Comment:
Just had a thought, we might not want to receive a whole host of alerts for every user error once we get alerting set up, so we could log these as warnings and have the option of handling them differently:

```suggestion
            console.warn(logEntry(event, HTTP_CODES.BAD_REQUEST, LOG_TYPES.ERROR, errorDetails));
```


(for example, another way you can trigger alerts in AWS is if a number of messages matching a filter you configure passes a certain threshold e.g. 10 in a minute - so we could for example choose to trigger an email/Slack message if there are a large number of warnings, which could signify a larger problem).